pom.xml

```diff
@@ -41,12 +69,23 @@
 				<artifactId>jackson-datatype-jsr310</artifactId>
 				<version>2.9.7</version>
 			</dependency>
-
+			<dependency>
+				<groupId>ca.szc.thirdparty.nl.jworks.markdown_to_asciidoc</groupId>
```

Comment:
siden dette er et bibliotek, trengs denne avhengigheten i det hele tatt eller kand et være opp til systemet som benytter? I såfall, hvis det fungerer å bygge og kompilere uten bør det kunne droppes (trenger ikke generere asciidoc her, holder med markup)

Reply:
Jeg vet ikke hvordan sysdoken genereres. For min del er det bare å fjerne og se om det funker

Reply:
Tenker vi kan ta ut det i en egen sak. Greit å få disse endringene inn.